pandas/core/window.py

```diff
@@ -11,6 +11,7 @@
 import numpy as np
 
 import pandas._libs.window as libwindow
+import pandas._libs._window as libwindow_refactor
```

Comment:
so I would either do this directly in .window or name this something non-private (as its already private). Note that if say refactoring needs to be done to make things cleaner we can do pre-cursors to master.